src/org/labkey/test/tests/AdminFolderTypeTest.java

```diff
@@ -0,0 +1,74 @@
+package org.labkey.test.tests;
+
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.categories.Daily;
+import org.labkey.test.components.ext4.RadioButton;
+import org.labkey.test.pages.core.admin.FolderTypePage;
+
+import java.util.List;
+
+@Category({Daily.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 2)
+public class AdminFolderTypeTest extends BaseWebDriverTest
+{
+    @Test
+    public void testDefaultFolderTypeSetting()
+    {
+        goToProjectHome();
+        String newDefaultFolder = "Targeted MS";
+
+        log("Setting a new default folder type");
+        FolderTypePage folderTypePage = goToAdminConsole().clickFolderType();
+        String oldDefaultFolder = folderTypePage.getDefaultFolder();
+        folderTypePage.setDefaultFolder(newDefaultFolder).clickSave();
+
+        folderTypePage = goToAdminConsole().clickFolderType();
+        checker().verifyEquals("Incorrect default Folder type selected", newDefaultFolder, folderTypePage.getDefaultFolder());
+
+        log("Verifying the default folder type while project creation");
+        goToCreateProject();
+        checker().verifyTrue("Incorrect default folder type selected",
+                RadioButton.RadioButton().withLabel("Panorama").find(getDriver()).isSelected());
+
+        log("Rollback to the old default folder type");
```

Comment:
Should this happen in a tear down method instead? If the test fails on line 33, for example, it will leave the default folder type as TargetedMS which seems likely to cause lots of other tests to fail.

Reply:
Sweta and I discussed this. There is, currently, no standard test code path that leaves the default folder type selected.